cmd/plugin/node_server.go

```diff
@@ -137,6 +149,8 @@ func (n NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublishV
 		Image:       image,
 		PullSecrets: req.Secrets,
 		Logger:      valuesLogger,
+		Pod:         req.VolumeContext[ctxKeyPodName],
+		Namespace:   req.VolumeContext[ctxKeyPodNamespace],
```

Comment:
Adding these might also help fix the bug in current logs where pod and namespace are empty in the log messages for printing image size.